src/components/Avatar/Avatar.tsx

```diff
@@ -110,6 +258,15 @@ export const Avatar: FC<AvatarProps> = React.forwardRef(
       minHeight: size,
       fontSize: fontSize,
       ...style,
+      ...(Object.keys(statusItems).length > 0 ? { position: 'relative' } : {}),
```

Comment:
this can be moved to a class

Reply:
`wrapperContainerStyle` is used at three places and creating a new class just for this one things would result in that new class to be added at all those places manually. And, it would be too much context switching in future to understand how styling is done to component.